lib/Radar/ValueListItem.php

```diff
@@ -17,10 +17,10 @@
  */
 class ValueListItem extends \Stripe\ApiResource
 {
-    const OBJECT_NAME = "radar.value_list_item";
+    const OBJECT_NAME = 'radar.value_list_item';
 
-    use \Stripe\ApiOperations\All;
     use \Stripe\ApiOperations\Create;
     use \Stripe\ApiOperations\Delete;
+    use \Stripe\ApiOperations\All;
```

Comment:
Nice catch but this needs a test

Reply:
Actually that was just the codegen tool moving the trait, not adding it. I've reverted the change.